### PR TITLE
Rewrite Vue charts code

### LINKED ISSUE #5089: Rewrite Vue charts code
Because the Vue charts library does not generate a release package, every time NPM install will pull the latest code again, which will have a certain impact on the stability of the front-end UI of this project.

~~Finally decided to replace the Vue charts Library to vue-echarts-v3 library~~

### PR DESCRIPTION
**Pull request by wqzwh**
Fixes #5089 .

Changes proposed in this pull request:
- add "resize-detector": "^0.2.1"
   This library is the one that the official Vue version of echarts relies on, so I also quoted
- add `shardingsphere-ui/shardingsphere-ui-frontend/src/components/ChartBase/index.vue`
   This is a secondary development, which is not completely consistent with the official code

### CODE REVIEW

**Comment by KomachiSion** [comment]
Can you add the licenses and notice information to LICENSE and NOTICE file?

**Comment by wqzwh** [comment]
> Can you add the licenses and notice information to LICENSE and NOTICE file?

sure，I'll add it now

File: shardingsphere-ui/shardingsphere-ui-distribution/shardingsphere-ui-bin-distribution/src/main/release-docs/NOTICE [hunk 1]
```diff
@@ -205,3 +205,15 @@ This product bundles icons from the famfamfam.com silk icons set
 http://www.famfamfam.com/lab/icons/silk/
 Licensed under the Creative Commons Attribution Licence v2.5
 http://creativecommons.org/licenses/by/2.5/
+
+========================================================================
+
+Apache echarts NOTICE
```
Comment (KomachiSion) [inline shardingsphere-ui/shardingsphere-ui-distribution/shardingsphere-ui-bin-distribution/src/main/release-docs/NOTICE:211]: should be `Apache ECharts (incubating) NOTICE`

Comment (wqzwh) [reply shardingsphere-ui/shardingsphere-ui-distribution/shardingsphere-ui-bin-distribution/src/main/release-docs/NOTICE:211]: ok

Comment (kimmking) [reply shardingsphere-ui/shardingsphere-ui-distribution/shardingsphere-ui-bin-distribution/src/main/release-docs/NOTICE:211]: line 211 should be incubating too

File: shardingsphere-ui/shardingsphere-ui-distribution/shardingsphere-ui-bin-distribution/src/main/release-docs/LICENSE [hunk 2]
```diff
@@ -242,6 +242,7 @@ The text of each license is the standard Apache 2.0 license.
     spring-framework 4.3.24.RELEASE: https://raw.githubusercontent.com/spring-projects/spring-framework, Apache 2.0
     tomcat 8.5.40: http://svn.apache.org/repos/asf/tomcat, Apache 2.0
     gson 2.8.5: https://github.com/google/gson, Apache 2.0
+    echarts: https://github.com/apache/incubator-echarts, Apache 2.0
```
Comment (KomachiSion) [inline shardingsphere-ui/shardingsphere-ui-distribution/shardingsphere-ui-bin-distribution/src/main/release-docs/LICENSE:245]: echarts licenses should follow groovy.

**Pull request merged**